common/djangoapps/third_party_auth/models.py

```diff
@@ -618,6 +618,13 @@ class SAMLProviderConfig(ProviderConfig):
             "This is helpful for testing/setup but should always be disabled before users start using this provider."
         ),
     )
+    country = models.CharField(
```

Comment:
I find that the EnterpriseCustomer model uses a countryField

from django_countries.fields import CountryField

I did not know about this 

https://github.com/edx/edx-enterprise/blob/05b320b190dffb072812546088ecf92aefbd63bc/enterprise/models.py#L14-L15

worth a look into why this vs string field

Reply:
interestingly when I look at the db table for enterprisecustomer, I see a VARCHAR(2)... lol .. certainly curious about this now

Reply:
I don't think this is relevant given this field shouldn't actually contain a country name.

Reply:
yup closing this one